ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   - blosc=1.21.0
   - bottleneck=1.3.2
   - brotlipy=0.7.0
-  - fastparquet=0.4.0
+  - fastparquet=0.6.3
```

Comment:
If you could follow up with changing this in `install.rst` that would be great.

Reply:
will do